com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/CollectionBuilder.java

```diff
@@ -454,6 +456,7 @@ protected CollectionDesc.Builder transform(Task task, IResource resource, Collec
         }
         messageBuilder.addAllPropertyResources(propertyResources);
 
+        compStorage.add("goc", goCount);
```

Comment:
Add `goCount` into storage, which will be summarized for each collection, so each collection will contain right counters